build-packages/scandipwa-cli/actions/deploy/index.js

```diff
@@ -33,7 +35,7 @@ const deploy = async (argv) => {
     // prepare application data
     const packageJson = getPackageJson(contextPathname);
     const appData = {
-        appId: packageJson.scandipwa.staticDeploy || '',
+        appId: appIdFromEnv || packageJson.scandipwa.staticDeploy || '',
```

Comment:
Can you please make a proper check for `staticDeploy` field, like if not set, show a proper `logger.error` for that? I also see no reason to move this variable into separate constant.

Reply:
Empty field means that application wasn't deployed yet. It's not an error.